packages/fx-core/templates/core/v3Migration/migrationReport.md

```diff
@@ -1,7 +1,19 @@
-# Teams Toolkit 5.0 Upgrade Summary
+# Upgrade Summary
 ## Migration overview
 
-## Detail change
+1. Upgrade projects from 2.X.X version to 3.0.0 version using migrators. 
```

Comment:
We should not mention V2 / V3 concepts in the markdown. They're just our internal concept.